### PR TITLE
Incoming

### PR DESCRIPTION
**Pull request by holland01**
Remarks provided in issue #4254.

### CODE REVIEW

File: src/library_workerfs.js [hunk 1]
```diff
@@ -15,8 +15,16 @@ mergeInto(LibraryManager.library, {
         var parent = root;
         for (var i = 0; i < parts.length-1; i++) {
           var curr = parts.slice(0, i+1).join('/');
+		  // Issue 4254: Using curr as a node name will prevent the node
```
Comment (kripken) [inline src/library_workerfs.js]: looks like indentation is wrong (are you using tabs instead of spaces?)

File: src/library_workerfs.js [hunk 2]
```diff
@@ -15,8 +15,16 @@ mergeInto(LibraryManager.library, {
...
+		  // from being found in FS.nameTable when FS.open is called on
+		  // a path which holds a child of this node,
+		  // given that all FS functions assume node names
+		  // are just their corresponding parts within their given path,
+		  // rather than incremental aggregates which include their parent's
+		  // directories.
           if (!createdParents[curr]) {
-            createdParents[curr] = WORKERFS.createNode(parent, curr, WORKERFS.DIR_MODE, 0);
+            createdParents[curr] = WORKERFS.createNode(parent, /*curr*/ parts[i],
+				WORKERFS.DIR_MODE, 0);
```
Comment (kripken) [inline src/library_workerfs.js]: same here. But actually please just put it on the same line and remove the `/*curr*/` part

**Comment by kripken** [comment]
Please also add yourself to AUTHORS.

**Comment by holland01** [comment]
Ok, should be good to go now. Let me know if there's anything else.

**Comment by kripken** [comment]
Thanks!

**Pull request merged**